MPF.Frontend/DumpEnvironment.cs

```diff
@@ -599,12 +599,21 @@ await Task.Run(() =>
                     else
                         resultProgress.Report(ResultEventArgs.Failure(compressResult));
 
+                    // Delete unnecessary files only after zipping
```

Comment:
In the call to `CompressLogFiles`, the `AddToArchive` method is called. There is where files are being deleted as it goes. The changes to this file should be unnecessary.